_data/places.yml

```diff
@@ -249,6 +236,19 @@ open:
       postmates: https://postmates.com/merchant/the-islands-brooklyn-1
       uber_eats: https://www.ubereats.com/new-york/food-delivery/the-islands/9jFnKcjVRBuUOpsVlzSgsg
     last_updated: 3/25
+ 
+  - name: James
+    url: http://www.jamesrestaurantny.com
+    instagram: https://www.instagram.com/jamesbklyn/
+    phone: (718) 942-4255
+    hours: 4 - 9PM
+    notes: “Call us at 718-942.4255 to place an order for take out and delivery available via Caviar.”
```

Comment:
Are you sure that they’ve reopened? Can’t find them on Caviar and not seeing any updates on Instagram 🤔

Reply:
They’re [on Caviar ](https://www.trycaviar.com/m/james-19693) now! Going to update the PR to include the link :)